src/LtiGrade.php

```diff
@@ -162,6 +162,8 @@ public function getCanvasExtension()
      *             is unexpected data. And, the type of LMS cannot simply be inferred by their URL.
      *
      * @see https://documentation.instructure.com/doc/api/score.html
+     *
+     * @todo: Deprecate this in the next major version
```

Comment:
I'm not certain about this one. Talking to Matt to see if this is something we actually want to do or not.